src/Tempest/Core/Kernel/LoadDiscoveryLocations.php

```diff
@@ -2,36 +2,37 @@
 
 declare(strict_types=1);
 
-namespace Tempest\Core\Bootstraps;
+namespace Tempest\Core\Kernel;
 
-use Tempest\Core\AppConfig;
+use Tempest\Core\DiscoveryException;
 use Tempest\Core\DiscoveryLocation;
+use Tempest\Core\Kernel;
 use Tempest\Support\PathHelper;
 
-final readonly class DiscoveryLocationBootstrap implements Bootstrap
+final readonly class LoadDiscoveryLocations
```

Comment:
TODO: mark these internal action classes as internal, they are just here so that Kernel isn't too crowded

Reply:
Or we refactor even further towards a more decoupled system